tools/config/datadog.yml

```diff
@@ -0,0 +1,5 @@
+title: Datadog Example Config
+order: 20
+backends:
+  - datadog-logs
+facets: []
```

Comment:
Can we add OOTB facets as defaults? e.g. `service` `source` `host` see https://docs.datadoghq.com/logs/explorer/facets/#out-of-the-box-facets

Reply:
We added all the default tags from https://docs.datadoghq.com/getting_started/tagging/#introduction in a21fe1e.